src/tiktok_uploader/browsers.py

```diff
@@ -80,24 +80,32 @@ def get_default_options(name: str, *args, **kwargs):
     raise UnsupportedBrowserException()
 
 
-def chrome_defaults(*args, headless: bool = False, **kwargs) -> ChromeOptions:
+def chrome_defaults(
+    *args, headless: bool = False, is_gcp: bool = False, **kwargs
+) -> ChromeOptions:
     """
     Creates Chrome with Options
     """
 
     options = ChromeOptions()
 
     ## regular
-    options.add_argument('--disable-blink-features=AutomationControlled')
-    options.add_argument('--profile-directory=Default')
+    options.add_argument("--disable-blink-features=AutomationControlled")
+    options.add_argument("--profile-directory=Default")
 
     ## experimental
-    options.add_experimental_option('excludeSwitches', ['enable-automation'])
-    options.add_experimental_option('useAutomationExtension', False)
+    options.add_experimental_option("excludeSwitches", ["enable-automation"])
+    options.add_experimental_option("useAutomationExtension", False)
 
     # headless
     if headless:
-        options.add_argument('--headless=new')
+        options.add_argument("--headless=new")
+        
+    if is_gcp:
```

Comment:
`is_gcp` should be a personal setting rather than a library feature. 

Honestly, maybe a "server mode" for when no display is present would be a good idea. I honestly only ran this on my Desktop and did not think anyone would actually run this on a cloud platform.